binding/binding_test.go

```diff
@@ -1064,6 +1080,24 @@ func testFormBindingInvalidName2(t *testing.T, method, path, badPath, body, badB
 	assert.Error(t, err)
 }
 
+func testFormBindingForCollectionFormat(t *testing.T, method, path, badPath, body, badBody string) {
+	b := Form
+	assert.Equal(t, "form", b.Name())
+
+	obj := FooStructForCollectionFormatTag{}
+	req := requestWithBody(method, path, body)
+	if method == "POST" {
+		req.Header.Add("Content-Type", MIMEPOSTForm)
+	}
+	err := b.Bind(req, &obj)
+	assert.NoError(t, err)
+
+	assert.Equal(t, []int{1, 2}, obj.SliceMulti)
```

Comment:
I'd refrain from using values 1 and 2 for each one of them. It can potentially conceal misbehaviour.